frontend/apps/crates/entry/jig/play/src/player/sidebar/dom/mod.rs

```diff
@@ -45,10 +46,15 @@ pub fn render(state: Rc<State>) -> Dom {
                 state.sidebar_open.set(false);
             }))
         }))
-        .child(html!("jig-play-sidebar-action", {
-            .property("slot", "actions")
-            .property("kind", "like")
-        }))
+        .child_signal(state.player_state.jig.signal_ref(clone!(state => move |jig| {
```

Comment:
Not sure that `published_at` is a reliable way to check if something is published, rather use `draftOrLive` to check.

Checking `published_at` used to be the preferred way to check if something is published but no longer is, so if you took this idea from somewhere else, then please create an issue for that.

Reply:
Oh great! I replicated the functionality for rendering the share button: https://github.com/johnnynotsolucky/ji-cloud/blob/sandbox/frontend/apps/crates/entry/jig/play/src/player/sidebar/dom/mod.rs#L52-L60

Will update that too.